src/services/message-bridge/index.ts

```diff
@@ -93,6 +96,13 @@ export class MessageBridge {
       return;
     }
 
+    if (type === MeetingEvent.MEETING_PARTICIPANT_JOINED) {
+      const { participants } = useStore(StoreType.GLOBAL);
+      const participantsList = { ...participants.value };
+      participantsList[data.id] = data;
```

Comment:
I think it's must be better to update the participant data instead of overwriting everything.

Reply:
You're right